2016-09-13 14:00:45.377  [E] [DBPoolX] Error: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2016-09-13 14:00:45.381  [E] [com.dao.LotoOnlineDao] getAll: Error executing >>>java.lang.NullPointerException
2016-09-13 14:03:43.442  [E] [DBPoolX] Error: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2016-09-13 14:04:14.553  [E] [com.dao.LotoOnlineDao] getAll: Error executing >>>java.lang.NullPointerException
2016-09-13 14:30:56.793  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.822  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.856  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.878  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.901  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.92  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.938  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.953  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.962  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.97  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.979  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.989  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:56.998  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.047  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.059  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.072  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.081  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.104  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.139  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.154  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.164  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.174  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.182  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.195  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.211  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.268  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.294  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.303  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.32  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.339  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.413  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.479  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.555  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.577  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.602  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.65  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.659  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.671  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.686  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:30:57.693  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, now(), ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.422  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.467  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.512  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.527  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.551  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.592  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.604  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.627  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.91  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.952  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:32.988  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.032  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.077  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.113  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.14  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.162  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.193  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.219  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.258  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.283  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.291  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.304  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.317  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.332  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.339  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.356  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.389  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.401  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.417  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.469  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.51  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.528  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.535  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.565  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.571  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.592  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.608  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.615  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.627  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
2016-09-13 14:32:33.636  [E] [com.dao.LotoOnlineDao] insertRow: Error executing SQL INSERT INTO capso_online(capso, sodiem, ngaychoi, gen_date, memberId, current_money) VALUES(?, ?, ?, ?, ?, ?)>>>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'memberId' in 'field list'
